### ISSUE TITLE
multiple clicks in the application breaks it down

### ISSUE THREAD

**Issue by ethael** [flutter/flutter#46069]
The same issue that is described in #45792 occurs again in current latest dev version: 1.12.16
version 1.12.15 does work as expected

**Comment by iapicca** [comment]
Hi @ethael 
yes, I'm experiencing the same.
To reproduce follow the [docs](https://flutter.dev/docs/get-started/web#summary)

[result](https://photos.app.goo.gl/LcNFS6aVYrDVidKd7)

<details>
  <summary>flutter doctor -v</summary>

```
[√] Flutter (Channel dev, v1.12.16, on Microsoft Windows [Version 10.0.19013.1122], locale en-US)
    • Flutter version 1.12.16 at C:\src\flutter
    • Framework revision 37f9c54116 (7 days ago), 2019-11-27 11:33:38 -0800
    • Engine revision fad1b23c42
    • Dart version 2.7.0


[√] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at C:\Users\iapic\AppData\Local\Android\sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 28.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b03)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Android Studio (version 3.5)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 41.1.2
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b03)

[√] VS Code, 64-bit edition (version 1.40.2)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension version 3.6.0

[√] Connected device (3 available)
    • Pixel 3a   • 965AY0WP5C • android-arm64  • Android 10 (API 29)
    • Chrome     • chrome     • web-javascript • Google Chrome 78.0.3904.108
    • Web Server • web-server • web-javascript • Flutter Tools

• No issues found!
```

</details>

**Comment by ferhatb** [comment]
Verified this is fixed in latest master:  1.12.17-pre.70
